src/ansys/units/systems.py

```diff
@@ -151,6 +145,11 @@ def solid_angle(self):
         """Solid Angle unit of the unit system."""
         return getattr(self, "_solid angle")
 
+    @staticmethod
+    def max_dim_len():
+        """Maximum number of elements within a dimensions list."""
+        return 10
```

Comment:
dimensions module / Dimensions class is the correct source for this information. We can terminate the enum with this constant and then it's always correct.